lib/package/commands.coffee

```diff
@@ -30,6 +30,14 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.toggleMeta 'methods'
+      'julia-client:reset-workspace': =>
+        if juno.connection.client.isActive()
+          editor = atom.workspace.getActiveTextEditor()
+          atom.commands.dispatch atom.views.getView(editor), 'inline-results:clear-all'
```

Comment:
Not sure we should mix up resetting the workspace and clearing the console. This will error if you're not in an editor anyway, I think.